mkdocs/docs/contribute.md

```diff
@@ -112,6 +135,13 @@ pip install pre-commit
 pre-commit install
 ```
 
+If you are using [`uv`](http://docs.astral.sh/uv/):
+
+```bash
+uv pip install pre-commit
+pre-commit install
+```
```

Comment:
I think `pre-commit` should be included in the `uv sync --all-extras` because it is listed in the `dev` dependencies. Thus, just `pre-commit install` should suffice here. but please test it locally.